lib/kramdown_rpf/rpf.rb

```diff
@@ -110,6 +112,31 @@ def self.convert_print_only_to_html(content)
         ::Kramdown::Document.new("<div class=\"u-print-only\">\n#{content}</div>", KRAMDOWN_OPTIONS).to_html
       end
 
+      def self.convert_quiz_to_html(quiz)
+        quiz =~ YAML_FRONT_MATTER_REGEXP
+        details = YAML.safe_load(Regexp.last_match(1))
+        question = details['question']
+        choices = YAML.safe_load(Regexp.last_match(2))
+        radios = choices.map { |c| c =~ RADIO_REGEXP ? $1 : c }.reject(&:empty?)
```

Comment:
Using `$1` is [not recommended](https://rubystyle.guide/#no-perl-regexp-last-matchers).  Again, I'd rather a named match, but I'm not sure it will fit in this context.